pkg/fleet/installer/setup/common/setup.go

```diff
@@ -119,6 +122,14 @@ func (s *Setup) Run() (err error) {
 	if err != nil {
 		return fmt.Errorf("failed to write install info: %w", err)
 	}
+	for _, group := range s.DdAgentAdditionalGroups {
+		// Add dd-agent user to additional group for permission reason
```

Comment:
you should be explicit with the case "in particular to enable reading log files not world readable", makes it easier to understand for next dev